package.json

```diff
@@ -21,7 +21,9 @@
   "homepage": "https://github.com/conlan/async-contracts#readme",
   "dependencies": {
     "@truffle/hdwallet-provider": "^1.0.29",
+    "ganache-cli": "^6.9.1",
     "openzeppelin-solidity": "^2.4.0",
+    "truffle": "^5.1.20",
```

Comment:
I added truffle and ganache-cli to package.json so that it can all just be installed together...